pkg/clusteragent/admission/mutate/tags.go

```diff
@@ -37,6 +37,13 @@ var labelsToEnv = map[string]string{
 	kubernetes.VersionTagLabelKey: kubernetes.VersionTagEnvVar,
 }
 
+var labelsToWorkloadEnv = [][2]string{
+	{kubernetes.ServiceTagLabelKey, "DD_WORKLOAD_SERVICE"},
+	{kubernetes.KubeAppNameLabelKey, "DD_WORKLOAD_SERVICE"},
+	{kubernetes.VersionTagLabelKey, "DD_WORKLOAD_VERSION"},
+	{kubernetes.KubeAppVersionLabelKey, "DD_WORKLOAD_VERSION"},
```

Comment:
Here I think you can have some conflict if a user set the 2 labels for the service but with 2 differents values, I think the `kubernetes.KubeAppNameLabelKey` will be selected a the end (due to slice iteration). But I think it can be better to get the precedence to the official Datadog labels key. 
(same remark for DD_WORKLOAD_VERSION).


Also I still think it is better to not introduce new specific `DD_WORKLOAD_SERVICE` and `DD_WORKLOAD_VERSION` but use the existing one. We can extend the current logic to look at `kubernetes.KubeAppNameLabelKey` if `kubernetes.ServiceTagLabelKey` it is something what already discussed in the past.

Reply:
If the user has set the 2 labels, the first one will 'win' (https://github.com/DataDog/datadog-agent/blob/main/pkg/clusteragent/admission/mutate/common.go#L77), that's why I let `ServiceTagLabelKey` before `KubeAppNameLabelKey`